eclair-core/src/main/scala/fr/acinq/eclair/router/Graph.scala

```diff
@@ -45,8 +45,8 @@ object Graph {
    * We use heuristics to calculate the weight of an edge based on channel age, cltv delta and capacity.
    * We favor older channels, with bigger capacity and small cltv delta.
    */
-  case class WeightRatios(cltvDeltaFactor: Double, ageFactor: Double, capacityFactor: Double) {
-    require(0 < cltvDeltaFactor + ageFactor + capacityFactor && cltvDeltaFactor + ageFactor + capacityFactor <= 1, "The sum of heuristics ratios must be between 0 and 1 (included)")
+  case class WeightRatios(biasFactor: Double, cltvDeltaFactor: Double, ageFactor: Double, capacityFactor: Double, hopCostBase: MilliSatoshi, hopCostMillionths: Long) {
```

Comment:
You can also update the comment on this class to explain that we favor shorter routes when the `biasFactor` (which needs to be renamed to mention what it biases on) is greater than 0 and `hopCostBase` and `hopCostMillionths` are as well.